tests/test_endpoint.py

```diff
@@ -111,11 +112,22 @@ def test_api_can_update_watering_information(self):
         next_water = last_watered.strftime("%a, %B %d %Y")
 
         res = self.client().post('/api/v1/garden/water?garden_plant_id={}'.format(gardenplant.id))
-        self.assertEqual(res.status_code, 200)
+        self.assertEqual(res.status_code, 201)
         self.assertIn("Dan", str(res.data))
         self.assertIn(next_water, str(res.data))
         self.assertIn(datetime.now().strftime("%a, %B %d %Y"), str(res.data))
 
+    def test_api_can_return_plant_profile_page(self):
+        garden = Garden(id=1)
+        adrian = Plant(id=5, plant_type='Buckcherry',image='adrian_photo.jpg',lighting='Full Sun',water_frequency=5,harvest_time=50,root_depth=12,annual="Annual")
+        gardenplant = GardenPlant(plant_id=adrian.id, garden_id=garden.id,plant_name="Adrian")
+        db.session.add_all([adrian, garden, gardenplant])
+        db.session.commit()
+
+        res = self.client().get('/api/v1/garden/plants/{}'.format(gardenplant.id))
```

Comment:
Nice job figuring this one out. I can't beieve all that we needed was to ad an ID to adrian